template/gem_name.gemspec.erb

```diff
@@ -38,7 +38,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rspec', '~> 3.0'
   spec.add_development_dependency 'simplecov', '~> 0.21.0'
 
-  spec.add_development_dependency 'rubocop', '~> 1.21.0'
```

Comment:
I recommend to separate this change to another PR.

Reply:
I've split this patch into #6